r2-netty/src/test/java/com/linkedin/r2/transport/http/client/TestHttpNettyStreamClient.java

```diff
@@ -1111,6 +1111,7 @@ public void onError(Throwable e)
                 latch.countDown();
               }
             }));
+        Thread.sleep(5);
```

Comment:
What is this for?

Reply:
This addresses a non-deterministic failure in the test, which happens even without the netty shading. It seems the Netty server gets overloaded by the requests, and we are relying on the retry to get past the failures, but sometimes the retry is not enough.

Reply:
Thanks for sharing the context. How about adding the information to the comment?

Reply:
Done